docs/src/validator/validator_length_rule.md

```diff
@@ -0,0 +1,20 @@
+## Length Rule
```

Comment:
As per our agreement for development this should be the first commit to this PR not the last